packages/fluentui/react-northstar-prototypes/src/prototypes/VirtualizedStickyTree/VirtualStickyTree.tsx

```diff
@@ -394,4 +401,5 @@ VirtualStickyTree.displayName = 'VirtualStickyTree';
 VirtualStickyTree.defaultProps = {
   accessibility: treeBehavior,
   itemSize: 50,
+  itemToString: item => (item as any).content || '',
```

Comment:
is `content` a  `ReactNode` here ? if so, might more sense to base it of `id`?

Reply:
also why `any`?

Reply:
It could be. I talked to shift about this function. The idea is to just have a dummy function, and have the user using virtualTree implement it. It's too complicated if we try to handle all situations ourselves.

Reply:
Let's probably not use a default value then ? or even just the empty string, since this is a prototype we don't want consumers who aren't too aware of React to think that `content` is the way to go, it could be dangerous

Reply:
Good point. Removed default prop